src/state.rs

```diff
@@ -2219,6 +2276,36 @@ impl PathDumpType {
     }
 }
 
+/// Returns the number of LLVM instructions in a passed path.
+/// The returned value is only accurate if the path under
+/// analysis does not include a panic, exception, exit,
+/// or any hooked calls (such as inline assembly).
+// A path is represented as a vector of `PathEntry`s, and
+// each `PathEntry` describes a sequential set of instructions in a basic block,
+// not necessarily starting at the beginning of that basic block.
+// When a `PathEntry` does not start at the beginning of a basic block, that means
+// we have returned to the basic block after a function call within that block.
+pub fn get_path_length<'p>(path: &Vec<PathEntry<'p>>) -> usize {
```

Comment:
I'm curious why you didn't make this a method on `State`?  (`get_path_length(&self)`)

Even if it's better to have this separate, I think we should also have a wrapper for it on `State` that just calls this

Reply:
I didn't make it a method on `State` because in my code using this PR I save several paths via `get_path()` but wasn't comparing them until later, so it was easier to save the `Path` instead of cloning the entire `State`. 

I could pretty easily refactor around this, but can generally imagine this being useful without the `State`. I'll add a wrapper.

Reply:
Wrapper added.